vendor/parser-extended/parsetree.mli

```diff
@@ -1113,7 +1116,7 @@ and value_bindings =
   {
     pvbs_bindings: value_binding list;
     pvbs_rec: rec_flag;
-    pvbs_extension: string loc option
+    pvbs_extension: string loc option;
```

Comment:
Unnecessary diff. We are careful with diffs in this file as we regularly have to backport changes to it.